### PR TITLE
Adds link step for xi deploys

### PR DESCRIPTION
**Pull request by willfish**
What

Link the frontend to all new backends so that we don't need to manually invoke the cf commands to link the routes.

Why

To avoid manual step when rolling deploys

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+cf 6.53.0
```
Comment (tavy88) [inline .tool-versions:1]: Any reason why we can't use >=7.0.0 here?

**Review by tavy88 — APPROVED** [review]
LGTM

File: .tool-versions (hunk 1 above)
Comment (willfish) [reply .tool-versions:1]: We'd need to update the bin/deploy script to follow the new syntax. One for later

**Review by mafonso — APPROVED** [review]
LGTM

**Pull request merged**